common-fragment-V1/src/main/java/org/pentaho/hadoop/shim/common/format/orc/PentahoOrcInputFormat.java

```diff
@@ -55,7 +57,11 @@ public PentahoOrcInputFormat( NamedCluster namedCluster ) {
       conf = inClassloader( () -> {
         Configuration confProxy = new ConfigurationProxy();
         confProxy.addResource( "hive-site.xml" );
-        ShimConfigsLoader.addConfigsAsResources( namedCluster.getName(), confProxy::addResource );
+        BiConsumer<InputStream, String> consumer = ( is, filename ) ->
+        {
+          confProxy.addResource( is, filename );
+        };
+        ShimConfigsLoader.addConfigsAsResources( namedCluster, consumer );
```

Comment:
Can we roll this functionality directly into the NamedCluster object and start to eliminate ShimConfigsLoader?

Reply:
If we put that in namedcluster than we add Vendor Library dependencies to the NamedCluster, which has versions that have to compile in the kettle.